tests/jobs/test_local_task_job.py

```diff
@@ -374,6 +374,29 @@ def test_localtaskjob_double_trigger(self):
 
         session.close()
 
+    @patch.object(StandardTaskRunner, 'return_code')
+    @mock.patch('airflow.jobs.scheduler_job.Stats.incr')
```

Comment:
It is good to have autospec=True parameter set for mocked objects, that will enforce method signature checks by unittest.mock library.
Although, I am not sure if this guideline is followed in the repo.

Reply:
Done.